build_tools/cmake/iree_third_party_cmake_options.cmake

```diff
@@ -48,6 +48,11 @@ macro(iree_add_llvm_external_project name identifier location)
   set(LLVM_EXTERNAL_PROJECTS ${LLVM_EXTERNAL_PROJECTS} CACHE STRING "" FORCE)
 endmacro()
 
+macro(iree_set_spirv_headers_cmake_options)
```

Comment:
I hate this file - it's the last place I'd look for settings despite the name - I really like having the settings near the FetchContents like we have with tint - maybe as we refactor things we can do that more. It also make sit easier to keep bazel and cmake in sync as the bzl and cmakelists live side-by-side instead of having all these things set in build_tools/cmake/. If we're able to fully exclude third_party/spirv_headers/ then maybe we switch that to FetchContent as well.